nc-times/src/components/ArticleById.jsx

```diff
@@ -10,20 +10,22 @@ function ArticleById() {
 
     const [loading, setLoading] = useState(true);
     const [articleId, setArticleId] = useState({})
+    const [like, setLike] = useState('')
     const { article_id } = useParams()
 
     useEffect(() => {
         setLoading(true)
         getArticleById(article_id)
             .then((articleData) => {
+                setLike(articleData.votes)
                 setLoading(false)
                 setArticleId(articleData)
             })
     }, [])
-    function clicker(){
-patchArticleById(article_id)
-.then((data)=>{
-    console.log(article_id)
+
+    function handleVote(vote){
+patchArticleById(article_id, vote).then((data)=>{
+setLike(data)
```

Comment:
consider doing this optimistically so the user can see immediate feedback of their like rather than waiting for the request to fulfil - see this: https://notes.northcoders.com/courses/js-front-end/ui-behaviours-and-optimistic-rendering#optimistic-rendering